src/interfaces/AdHocMeasureTitle.ts

```diff
@@ -0,0 +1,24 @@
+// (C) 2007-2018 GoodData Corporation
+/**
+ * The interface of the object used by the {ArithmeticMeasureTitleFactory} to not be dependent on the
+ * {VisualizationObject.IMeasure}. It contains all the necessary properties related to the measure title.
+ *
+ * @internal
+ */
+export interface IMeasureTitleProps {
+    localIdentifier: string;
+    title?: string;
+    alias?: string;
+}
+
+/**
+ * The interface of the object used by the {ArithmeticMeasureTitleFactory} to not be dependent on the
+ * {VisualizationObject.IArithmeticMeasureDefinition}. It contains all the necessary properties from the arithmetic
+ * measure.
+ *
+ * @internal
+ */
+export interface IArithmeticMeasureProps {
```

Comment:
can we name it something like `IArithmeticMeasureTitleProps`

Reply:
Fixed.